src/frontend/test_runner/tests/testdata/nexmark.yaml

```diff
@@ -187,17 +188,18 @@
               StreamProject { exprs: [$1, $3, $0] }
                 StreamHashAgg { group_keys: [$0, $1], aggs: [count, count] }
                   StreamExchange { dist: HashShard([0, 1]) }
-                    StreamProject { exprs: [$3, $0, $2] }
-                      StreamHopWindow { time_col: $1 slide: 00:00:02 size: 00:00:10 }
-                        StreamTableScan { table: bid, columns: [auction, dateTime, _row_id], pk_indices: [2] }
+                    StreamProject { exprs: [$1, $0, $2] }
```

Comment:
seems that some extra project nodes are generated in stream/batch plan :thinking:

Reply:
strange to me. they should be merged by `ProjectMergeRule`

Reply:
This might be because the project merge rule only applies to logical plan, and the extra stream project here is generated when converting the optimized logical plan to stream plan.

Reply:
Should we apply `ProjectMergeRule` again here?